Sources/Core/Presenter/Channel/ChannelPresenter.swift

```diff
@@ -200,7 +201,7 @@ extension ChannelPresenter {
                               attachments: attachments,
                               mentionedUsers: mentionedUsers,
                               extraData: extraData,
-                              showReplyInChannel: false)
+                              showReplyInChannel: showReplyInChannel && parentId != nil)
```

Comment:
We use `parentId != nil` a lot all over the place. Maybe we can extract it to a more meaningful variable like `isThread` ?